documentation/modules/configuring/proc-config-mirrormaker2-replication.adoc

```diff
@@ -54,6 +54,8 @@ This procedure shows a configuration that uses TLS encryption and authentication
 ** link:{BookURLDeploying}#deploying-kafka-cluster-str[Kafka cluster^]
 * Source and target Kafka clusters must be available
 
+WARNING: To avoid conflicts between dependencies, MirrorMaker 2.0 and the underlying Kafka Connect deployment that supports it must be from the same Kafka version.
```

Comment:
I do not think this really should be a warning. There is no real way to do it. So I think we just need to clarify it to not get too many questions about it.

Reply:
removed